v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/MigrateTableTransform.java

```diff
@@ -107,7 +107,7 @@ public PCollection<Void> expand(PBegin input) {
       outputDirectory += "/";
     }
     // Dump Failed rows to DLQ
-    String dlqDirectory = outputDirectory + "dlq/" + shardId;
```

Comment:
There's no UT for this class yet. `MigrateTableTransform` spawns the entire pipeline.